en/SUMMARY.md

```diff
@@ -117,33 +145,8 @@
   * [Geofence](flying/geofence.md)
   * [Safety Point Planning](flying/plan_safety_points.md)
   * [Flight Modes](flight_modes/README.md)
```

Comment:
This is the "table summary". Probably just a link to the other docs. Not sure yet, we still have a few "general" docs undeneath.

Reply:
I removed the big table, replacing with links to vehicle specific docs. The content of the table is in each of the detail pages.